content/en/blog/_posts/2022-12-01-scalable-job-tracking-ga/index.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: blog
+title: "Scalable Job tracking goes GA to support massively parallel batch workloads"
+date: 2022-12-01
+slug: "scalable-job-tracking-ga"
+---
+
+**Authors:** Aldo Culquicondor (Google)
+
+The Kubernetes 1.26 release includes a stable implementation of the Job
+controller that can reliably track a large amount of Jobs with high levels of
+parallelism. SIG Apps have worked on this foundational improvement, under
+the feature gate `JobTrackingWithFinalizers`, since Kubernetes 1.22. After
```

Comment:
I'd avoid mentioning the feature gate since it's not really relevant after the GA